src/components/Transactions/ScheduleTransactionForm.js/index.js

```diff
@@ -34,34 +34,33 @@ class ScheduleTransactionForm extends React.Component {
     handleSubmit = async (event) => {
         event.preventDefault();
         const timeFormated = moment(this.state.transaction_time).format('YYYY-MM-DDTHH:mm:ss.00000ZZ')
-        const newState = {...this.state, transaction_time: timeFormated}
+        const newState = { ...this.state, transaction_time: timeFormated }
         const res = await API.createScheduledTransaction(newState)
         if (res.status === 201) {
-            alert ("Transaction Scheduled")
-            this.props.scheduledTransactionHandler()
+            alert("Transaction Scheduled");
+            event.target.parentNode.reset();
```

Comment:
why are we resetting an element's value by doing this instead of resetting its state